Automattic/woocommerce-payments
# Fix Link button alignment in the Checkout block

Fixes #7929 

#### Changes proposed in this Pull Request

This ensures that the Stripe Link button is correctly aligned to the middle of the email input when viewing the Checkout block.









**Before**
<img width="400" src="https://private-user-images.githubusercontent.com/82999806/291453380-1be3f30d-3bfb-42a6-bc23-44db28b8382b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ3NDQyMTQsIm5iZiI6MTcwNDc0MzkxNCwicGF0aCI6Ii84Mjk5OTgwNi8yOTE0NTMzODAtMWJlM2YzMGQtM2JmYi00MmE2LWJjMjMtNDRkYjI4YjgzODJiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTA4VDE5NTgzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMwMDViZDM3NzgxMWNiNWY1OTBjZDQ2YmE1MzU0ZWI4ZTIxMDU0MjdjMWUxOWU1NDdkYWZmZDlkMGZlODNmOGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._l0gSO_IcCbSjaGbpzD6inZC2UdkfrcQ2rCZ0BJRAUU" />
**After**
<img width="400" src="https://github.com/Automattic/woocommerce-payments/assets/3473953/5c57eebb-bef4-4a4d-ab3d-d05f4ca529b8" />


#### Testing instructions

1. Activate Link by Stripe.
2. Ensure that the Checkout block is in use on the site.
3. Log into an account that already has an existing Link account (see https://woo.com/document/woopayments/payment-methods/link-by-stripe/#logged-in-user-experience)
4. Navigate to the checkout page.
5. Check that the Stripe Link button is vertically aligned to the middle on the email input

-------------------


**Post merge**
